src/test/java/io/stargate/sgv2/jsonapi/api/v1/AbstractCollectionIntegrationTestBase.java

```diff
@@ -49,6 +49,35 @@ protected void createCollection(String collectionToCreate) {
         .statusCode(200);
   }
 
+  protected void createCollection(String collectionToCreate, String indexingConfig) {
```

Comment:
Almost, but I think it should be passed JSON Object that is value of `"createCollection"`, otherwise hard-codes vector settings. So instead of "name", pass "collectionJson" String or so.